cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -253,7 +261,7 @@ endfunction ()
 # Most of them are subdirectories in `google/cloud/`. Some number of them have
 # additional samples that are enabled if needed.
 function (google_cloud_cpp_enable_features)
-    foreach (feature ${GOOGLE_CLOUD_CPP_ENABLE})
+    foreach (feature IN LISTS ${GOOGLE_CLOUD_CPP_ENABLE})
```

Comment:
If you use `IN LISTS` you want the name of the lists, not their values, i.e. you want `GOOGLE_CLOUD_CPP_ENABLE`, not  `${GOOGLE_CLOUD_CPP_ENABLE}`

Reply:
:monkey: 

D'oh. Thanks.